source/dlangbot/app.d

```diff
@@ -393,6 +401,40 @@ void setBotTimer(C)(Duration dur, C callback)
         callback();
 }
 
+// compute the number of points the author will get for fixing any issues
+private uint getTotalPointsPerPR(T)(T bugzillSeverities)
+if (isInputRange!T)
+{
+    import std.algorithm : map, sum;
+    uint severityToPoints(string severity)
+    {
+        switch(severity)
+        {
+            case "enhancement", "trivial":
+                return 10;
+            case "minor":
+                return 15;
+            case "normal":
+                return 20;
+            case "major":
+                return 50;
+            case "critical", "blocker":
+                return 75;
+            case "regression":
+                return 100;
+            default: assert(0);
```

Comment:
This will completely crash the bot should we change the Bugzilla configuration to add a severity. Not sure if this is a good thing, perhaps logging an error and returning 0 may be better.